lighthouse-cli/bin.ts

```diff
@@ -241,8 +242,15 @@ function showProtocolTimeoutError() {
   process.exit(_PROTOCOL_TIMEOUT_EXIT_CODE);
 }
 
+function showPageLoadError() {
+  console.error('Unable to load the page.');
+  process.exit(_PAGE_LOAD_ERROR_EXIT_CODE);
```

Comment:
nope, no reason. Just trying to match the style of the surrounding code. Will change to `_RUNTIME_ERROR_CODE`